error-issue/test/bot.test.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import nock from 'nock';
```

Comment:
@estherkim used jest to mock Octokit, which, after I saw that, made a lot more sense to me than using `nock`. I think we should adopt that as our preferred method of testing for GitHub API actions

Reply:
I don't really agree there, especially when testing flows with multiple calls (this app will include GitHub GraphQL, GitHub Rest, and GCloud logging API calls). I see more value in testing that the expected requests are being made (and failing tests where unexpected requests are made) rather than tying into the Octokit/library methods. We use `nock` for testing in every app except `pr-deploy` and I'd prefer to stick with this, which works well.

Reply:
Discussed offline, resolving for now, discussion around migration away to happen in the future.